packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -467,9 +474,14 @@ const DropdownMenuSubContent = React.forwardRef<
       ref={forwardedRef}
       style={{
         ...props.style,
-        // re-namespace exposed content custom property
-        ['--radix-dropdown-menu-content-transform-origin' as any]:
```

Comment:
This was exposed as `…-dropdown-menu-content-…`, even though it's the sub content. I've done the same for now, but I'm wondering whether it was an omission or on purpose, I can see a reason for both…